runtimes/eden/src/xcm_config.rs

```diff
@@ -189,22 +186,30 @@ impl pallet_xcm::Config for Runtime {
 	type MaxLockers = ConstU32<8>;
 	type AdminOrigin = EnsureRoot<AccountId>;
 	type WeightInfo = crate::weights::pallet_xcm::WeightInfo<Runtime>;
-	#[cfg(feature = "runtime-benchmarks")]
-	type ReachableDest = ReachableDest;
 	type MaxRemoteLockConsumers = ConstU32<0>;
 	type RemoteLockConsumerIdentifier = ();
 }
 
+parameter_types! {
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = Concrete(NodlLocation::get());
+	/// The base fee for the message delivery fees.
+	pub const BaseDeliveryFee: u128 = crate::constants::NODL.saturating_mul(3);
```

Comment:
@ETeissonniere do you have any intuition what could be a fair fee here?

Reply:
[Other projects][1] use either `3_000_000_000` or` CENTS.saturating_mul(3);` I will align our code to this as well.


[1]:  https://github.com/search?q=%22pub+const+BaseDeliveryFee%3A%22+language%3ARust&type=code